### PR TITLE
fix(parser): Query parser now accept boolean and number values

### PR DESCRIPTION
**Pull request by salomaosnff**
Fix of my issue #97

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,9 +16,9 @@
     "pretest:coveralls": "yarn pretest",
     "test:coveralls": "yarn s test.coveralls",
     "start:typeorm": "npx nodemon -w ./integration/crud-typeorm -e ts node_modules/.bin/ts-node integration/crud-typeorm/main.ts",
-    "db:sync:typeorm": "cd ./integration/crud-typeorm && npx ts-node -r tsconfig-paths/register ../../node_modules/.bin/typeorm schema:sync -f=orm",
-    "db:drop:typeorm": "cd ./integration/crud-typeorm && npx ts-node -r tsconfig-paths/register ../../node_modules/.bin/typeorm schema:drop -f=orm",
-    "db:seeds:typeorm": "cd ./integration/crud-typeorm && npx ts-node -r tsconfig-paths/register ../../node_modules/.bin/typeorm migration:run -f=orm",
+    "db:sync:typeorm": "cd ./integration/crud-typeorm && npx ts-node -r tsconfig-paths/register ../../node_modules/typeorm/cli.js schema:sync -f=orm",
+    "db:drop:typeorm": "cd ./integration/crud-typeorm && npx ts-node -r tsconfig-paths/register ../../node_modules/typeorm/cli.js schema:drop -f=orm",
+    "db:seeds:typeorm": "cd ./integration/crud-typeorm && npx ts-node -r tsconfig-paths/register ../../node_modules/typeorm/cli.js migration:run -f=orm",
```
Comment (zMotivat0r) [inline package.json]: what's the benefit of this change?

File: packages/crud-request/package.json [hunk 2]
```diff
@@ -46,5 +46,8 @@
   "scripts": {
     "prebuild": "npx rimraf ./lib",
     "build": "tsc -b"
+  },
+  "dependencies": {
```
Comment (zMotivat0r) [inline packages/crud-request/package.json]: why would we need dependencies here?
the core `crud` package has `util` as a dependency already

File: packages/crud-typeorm/package.json [hunk 3]
```diff
@@ -38,5 +38,9 @@
   "scripts": {
     "prebuild": "npx rimraf ./lib",
     "build": "tsc -b"
+  },
+  "dependencies": {
```
Comment (zMotivat0r) [inline packages/crud-typeorm/package.json]: same here. I think it's better to add those as a `peerDependencies`

File: package.json (hunk 1 above)
Comment (salomaosnff) [reply package.json]: I was not able to run these commands without Windows system. And with this change I did it.

File: packages/crud-request/package.json (hunk 2 above)
Comment (salomaosnff) [reply packages/crud-request/package.json]: When building the project with the command yarn s build, I received modules errors not found, placing a reference in package.json, lerna was able to link these dependencies and I was able to build and run the tests.

File: packages/crud/package.json [hunk 4]
```diff
@@ -40,8 +40,6 @@
     "build": "tsc -b"
   },
   "dependencies": {
-    "@nestjsx/crud-request": "^4.0.0",
```
Comment (zMotivat0r) [inline packages/crud/package.json]: No need to remove those deps here.

File: packages/crud-request/package.json (hunk 2 above)
Comment (zMotivat0r) [reply packages/crud-request/package.json]: > When building the project with the command yarn s build, I received modules errors not found, placing a reference in package.json, lerna was able to link these dependencies and I was able to build and run the tests.

please do rebase.
remove all your `npm_modules`
run `yarn bootstrap`

Comment (zMotivat0r) [reply packages/crud-request/package.json]: I fixed `yarn build`

**Pull request merged**